src/InteractiveWindow/Editor/Commands/InteractiveWindowCommands.cs

```diff
@@ -166,7 +167,9 @@ public IEnumerable<IInteractiveWindowCommand> GetCommands()
 
         internal IEnumerable<string> Help()
         {
-            string format = "{0,-" + _maxCommandNameLength + "}  {1}";
+            // The magic number `19` here is calculated based on how the description is dispayed for other help entries to keep the texts aligned
+            // (As of now other help entries include REPL commands and script derectives, both have manually formatted help description strings.
+            string format = "{0,-" + Math.Max(19, _maxCommandNameLength) + "}  {1}";
```

Comment:
I also fixed the text alignment for REPL commands description, which really bothers me. Please let me know if you think using magic# is evil.

![snip_20160203160906](https://cloud.githubusercontent.com/assets/788783/12801319/aa620ace-ca90-11e5-9ac4-c25a3c1278e2.png)